controllers/ptpoperatorconfig_controller.go

```diff
@@ -214,7 +214,15 @@ func (r *PtpOperatorConfigReconciler) syncLinuxptpDaemon(ctx context.Context, de
 		}
 	}
 
-	enabledPlugins := strings.Join(defaultCfg.Spec.EnabledPlugins, ",")
+	var pluginList []string
+
+	if defaultCfg.Spec.EnabledPlugins != nil {
+		pluginList = *defaultCfg.Spec.EnabledPlugins
+	} else {
+		pluginList = []string{"e810"} // Enable e810 by default if plugins not specified
```

Comment:
can we have e810 as  map  in api types, so in future usecase ptpdaemon can reference from there?
[string]string{}

Reply:
Think of operator upgrades when you design to depreciate any variables